conf/index-settings.json

```diff
@@ -5,8 +5,9 @@
 			"analysis": {
 				"analyzer": {
 					"default": {
-						"tokenizer": "standard",
-						"filter": ["lowercase", "german_filter"]
+						"type" : "custom",
+						"tokenizer" : "my_hyphen_tokenizer",
```

Comment:
The "my_" prefix seems a little weird. How about just "hyphen_tokenizer"?

(Same for the old "my_edge_ngram_tokenizer" that I probably introduced myself).